### ISSUE TITLE
Infinite loading always loading up content from index, not tag or author pages

### ISSUE THREAD

**Issue by tobimori** [styxlab/gatsby-theme-try-ghost#53]
Hi there.
I'm currently in the works of porting our old Ghost theme to Gatsby. While doing that, I stumbled upon your packages.

I've implemented Infinite Scrolling on my theme (which can be found [here](https://github.com/shorttechde/ingenious-gatsby) - a live version is available [here](https://shorttech.netlify.app) and noticed, that the infinite scrolling on the author and tag pages just uses the index content.

You can compare it with [this instance](https://shorttech.de/), running our old, Ghost-based theme. The password is `relaunch`.

**Comment by styxlab** [comment]
Hi @tobimori, thanks for reporting this issue and for bringing your work to my attention - it looks really nice.

Yes there is an issue with the author and tag pages when using infinite scroll. I'd be happy to take an PR, if you want to fix this.

**Comment by tobimori** [comment]
I'm happy If you do it 😄

**Comment by styxlab** [comment]
This is actually a bit tricky. I haven't quite made up my mind on how to best solve this.

**Comment by tobimori** [comment]
On our old Ghost Theme we fetched the contents from the next site and just added the contents of the Post List there to the Post List on the main site

```
        let page = 2;
        let url_blog = window.location.href.replace("\/#", "/");
        let ids = ['p31', 'p32', 'p33', 'p21', 'p22', 'p1']
        let max_page = {{ pagination.pages }};
        let currentlyLoading = false;
        function ready(fn) {
            if (document.readyState != 'loading') {
                fn();
            } else {
                document.addEventListener('DOMContentLoaded', fn);
            }
        }
        function loadContent() {
            var req = new XMLHttpRequest();
            req.onreadystatechange = function () {
                if (this.readyState == 4 && this.status == 200) {
                    parser = new DOMParser();
                    doc = parser.parseFromString(this.responseText, "text/html");
                    ids.forEach((id) => {
                        document.getElementById(id).innerHTML += doc.getElementById(id).innerHTML;
                    })
                    currentlyLoading = false;
                }
            };
            req.open("GET", url_blog + 'page/' + page, true);
            req.send();
            page = page + 1;
        }
        function bottomDistance() {
            return Math.max(document.body.offsetHeight - (window.pageYOffset + window.innerHeight), 0);
        }
        ready(() => {
            document.addEventListener('scroll', function () {
                if (bottomDistance() < 1000 && !currentlyLoading) {
                    currentlyLoading = true;
                    loadContent();
                }
            })
        })
```

**Comment by styxlab** [comment]
@tobimori: really interesting idea. For now I will stick to the current mechanism of fetching posts from files, but would reconsider if you method proofs faster.

**Comment by styxlab** [comment]
Fixed in npm version `>=1.10.2`.
